### PR TITLE
enh(release): Prepare Centreon Web 20.10.8 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,43 @@ If you have feature requests or want to report a bug, please go to our
...
+
+- [Administration] Broker statistics for pollers are not shown
+- [APIv2] DELETE downtime on host not functionnal
+- [Configuration] Change default values for Centreon Engine
+- [Configuration] Unable to add a Poller with the Wizard
+- [Core] Avoid 404 redirection
+- [Resources Status] "Filter by Host" filter is not emptied between searches
+- [Resources Status] Apply ACL in command line block
+- [Resources Status] Manage not filled curves
+
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:34]: Missing: https://centreon.atlassian.net/browse/MON-7368

File: fr/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,43 @@ commerciales, vous pouvez vous rendre sur notre
...
+
+- [Administration] Broker statistics for pollers are not shown
+- [APIv2] DELETE downtime on host not functionnal
+- [Configuration] Change default values for Centreon Engine
+- [Configuration] Unable to add a Poller with the Wizard
+- [Core] Avoid 404 redirection
+- [Resources Status] "Filter by Host" filter is not emptied between searches
+- [Resources Status] Apply ACL in command line block
+- [Resources Status] Manage not filled curves
+
```
Comment (thiuyendang) [inline fr/releases/centreon-core.md:34]: Missing: https://centreon.atlassian.net/browse/MON-7368

File: fr/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,43 @@ commerciales, vous pouvez vous rendre sur notre
...
+- [Configuration] SQL Injection on poller form
+- [Configuration] Unserialize() are not sanitized in Centreon Broker wizard
+- [Configuration] Unserialize() are not sanitized in poller wirard
+- [Configuration] XSS reflected on Graph performance curves
+- [Configuration] XSS reflected on SNMP trap
+- [Configuration] XSS reflected on internal API broker configuration
+- [Graph] SQL Injection on Graph component templates
+- [Graph] SQL Injection on Graph generate image
+- [Reporting] SQL Injection on reporting export
+
```
Comment (thiuyendang) [inline fr/releases/centreon-core.md:53]: Missing: https://centreon.atlassian.net/browse/MON-7199

File: en/releases/centreon-core.md [hunk 4]
```diff
@@ -19,6 +19,43 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] SQL Injection on poller form
+- [Configuration] Unserialize() are not sanitized in Centreon Broker wizard
+- [Configuration] Unserialize() are not sanitized in poller wirard
+- [Configuration] XSS reflected on Graph performance curves
+- [Configuration] XSS reflected on SNMP trap
+- [Configuration] XSS reflected on internal API broker configuration
+- [Graph] SQL Injection on Graph component templates
+- [Graph] SQL Injection on Graph generate image
+- [Reporting] SQL Injection on reporting export
+
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:53]: Missing: https://centreon.atlassian.net/browse/MON-7199

File: en/releases/centreon-core.md [hunk 5]
```diff
@@ -19,6 +19,43 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Administration] Import of JS in image files
+- [Administration] Insecure media file upload
+- [Administration] SQL Injection on ACL actions
+- [Administration] SQL Injection on ACL resources
+- [Administration] SQL Injection on reload ACL
+- [Configuration] SQL Injection on MediaWiki
+- [Configuration] SQL Injection on SNMP trap manufacturer
+- [Configuration] SQL Injection on poller form
+- [Configuration] Unserialize() are not sanitized in Centreon Broker wizard
+- [Configuration] Unserialize() are not sanitized in poller wirard
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:46]:
```suggestion
- [Configuration] Unserialize() are not sanitized in poller wizard
```

File: fr/releases/centreon-core.md [hunk 6]
```diff
@@ -19,6 +19,43 @@ commerciales, vous pouvez vous rendre sur notre
...
+- [Administration] Import of JS in image files
+- [Administration] Insecure media file upload
+- [Administration] SQL Injection on ACL actions
+- [Administration] SQL Injection on ACL resources
+- [Administration] SQL Injection on reload ACL
+- [Configuration] SQL Injection on MediaWiki
+- [Configuration] SQL Injection on SNMP trap manufacturer
+- [Configuration] SQL Injection on poller form
+- [Configuration] Unserialize() are not sanitized in Centreon Broker wizard
+- [Configuration] Unserialize() are not sanitized in poller wirard
```
Comment (thiuyendang) [inline fr/releases/centreon-core.md:46]:
```suggestion
- [Configuration] Unserialize() are not sanitized in poller wizard
```

**Review by thiuyendang — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

**Review by thiuyendang — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 7]
```diff
@@ -19,6 +19,53 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.10.8
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:23]:
```suggestion

`June 7, 2021`

```

File: fr/releases/centreon-core.md [hunk 8]
```diff
@@ -19,6 +19,53 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.10.8
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:23]:
```suggestion

`7 Juin 2021`

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**